specs/http-client/index.html

```diff
@@ -429,117 +428,115 @@ <h3>Serializing the request content</h3>
           the content is <emph>tidied up</emph> and parsed (this process is
-          implementation-dependant) and the item is the resulting document node.  If this is a
-          binary media type, the content is returned as a base64Binary item.  From the previous
-          rules, a result item can then be either a document node (from XML or HTML), a string or a
-          base64Binary.</p>
+          implementation-defined) and the item is the resulting XDM <code>document-node()</code>.  If this is a
+          binary media type, the content is returned as an <code>xs:base64Binary</code> item.  From the previous
+          rules, a result item can then be either a <code>document-node()</code> (from XML or HTML), an <code>xs:string</code>, or a
+          <code>xs:base64Binary</code>.</p>
         <p>When the type of a part is either XML or HTML, its body has to be parsed into a document
-          node. If there is any error when parsing the content, an error is raised with an
-          appropriate message <a href="#dfn-errHC002">err:HC002</a>.</p>
+          node. If an error occurs whilst parsing the content, the error <a>err:HC002</a> MUST be raised.</p>
         <p>If the attribute <code>override-media-type</code> is set on the request, its value is
-          used instead of the Content-Type returned by the HTTP server. If the Content-Type of the
-          response is a multipart type, the value of <code>override-media-type</code> can only be a
+          used instead of the <code>Content-Type</code> header returned by the HTTP server. If the <code>Content-Type</code> header of the
+          response indicates a multipart type, the value of <code>override-media-type</code> can only be a
           multipart type, or <code>application/octet-stream</code> (to get the raw entity as a
-          binary item). If it is not, this is an error <a href="#dfn-errHC003">err:HC003</a>.</p>
+          binary item). If it is not, the error <a>err:HC003</a> MUST be raised.</p>
       </section>
     </section>
 
     <section>
-      <h2>Content types handling</h2>
-      <p>In both requests and responses, MIME type strings are used to choose the way the entity
-        content has to be respectively serialized or parsed.  Four different kinds of type are
-        defined here, which are used in the above text about sending request and receiving response.
-        The intent is to provide the spirit of the entity content handling regarding its content
-        type, but an implementation is encouraged to deviate from those rules if it is obvious that
-        a particular type should be treated in a specific way (normally, that would be the case only
-        to treat a binary type as another type).</p>
+      <h2>Processing Media Types</h2>
+      <p>In both requests and responses, Internet Media Type strings are used to choose the way the entity
```

Comment:
I don't see the phrase "Internet Media Type" in https://tools.ietf.org/html/rfc6838. Consider just "Media Type" here and in subsequent invocations.